docs/howto/openstack/nova/move-server-between-regions.md

```diff
@@ -0,0 +1,221 @@
+| description         | None                                 |
+| encrypted           | False                                |
+| id                  | 9e171aa3-....-....-....-............ |
+| multiattach         | False                                |
+| name                | <new_volume_name>                    |
+| properties          |                                      |
+| replication_status  | None                                 |
+| size                | 20                                   |
+| snapshot_id         | None                                 |
+| source_volid        | 14942b23-....-....-....-............ |
+| status              | creating                             |
+| type                | default                              |
+| updated_at          | None                                 |
+| user_id             | ................................     |
++---------------------+--------------------------------------+
+```
+
+## Creating an image of a volume
+
+Then create the image, by using the following command:
+
+```bash
+openstack image create --volume <volume_id> <new_image_name>
+```
+
+> Substitute `<volume_id>` with the ID from the newly created volume in the previous step.
+
+After a while you will get a printout showing you information of the new image, such as the current status and the image ID. Depending on the size of the volume it might take a while to upload and while it is the image `status` will be `uploading`
+
+```plain
++---------------------+--------------------------------------+
+| Field               | Value                                |
++---------------------+--------------------------------------+
+| container_format    | bare                                 |
+| disk_format         | raw                                  |
+| display_description | None                                 |
+| id                  | 9e171aa3-....-....-....-............ |
+| image_id            | de7e1fcc-....-....-....-............ |
+| image_name          | <new_image_name>                     |
+| protected           | False                                |
+| size                | 20                                   |
+| status              | uploading                            |
+| updated_at          | 2022-11-10T15:22:12.000000           |
+| visibility          | shared                               |
+| volume_type         | default                              |
++---------------------+--------------------------------------+
+```
+
+Before you continue, make sure the image `status` field is `active`, otherwise wait a bit and then check again with:
+
+```bash
+openstack image show -c status <image_id>
+```
+
+The printout should look like this before you continue.
+
+```plain
++--------+--------+
+| Field  | Value  |
++--------+--------+
+| status | active |
++--------+--------+
+```
+
+> It is now safe to remove the volume you created earlier.
```

Comment:
Add a command for this. Again, make sure you use the placeholder `<copy_volume_id>`, not `<volume_id>`.